app/src/js/components/Datepicker/Datepicker.js

```diff
@@ -63,6 +63,7 @@ class Datepicker extends React.PureComponent {
 
   componentDidMount () {
     updateDatepickerStateFromQueryParams(this.props);
+    this.homePageInitialDateTime();
```

Comment:
I think this is probably going to get removed.

Reply:
I updated it a little bit, but should it be combined with `updateDatepickerStateFromQueryParams`? Although maybe it makes sense to keep their functions separate

Reply:
This is fine.